### DISCUSSION TITLE
The version rang of '@swc/helpers' in the latest version's peerDependencies does not exist

### DISCUSSION THREAD

**Discussion by easesu** [swc-project/swc#7204] [General]
@swc/core latest version is 1.3.45, its peerDependencies @swc/helpers require a version range ">0.4.14 <0.5", see [here](https://www.npmjs.com/package/@swc/core/v/1.3.45?activeTab=code)
but the required version range does not exist, see [here](https://www.npmjs.com/package/@swc/helpers/v/0.4.13?activeTab=versions)

**Comment by zymon** [comment]
The same is for ts-node:
`npm WARN ERESOLVE overriding peer dependency`
`npm WARN While resolving: @swc/core@1.3.45`
`npm WARN Found: @swc/helpers@undefined`
`npm WARN node_modules/@swc/helpers`
`npm WARN`
`npm WARN Could not resolve dependency:`
`npm WARN peer @swc/helpers@">0.4.14 <0.5" from @swc/core@1.3.45`
`npm WARN node_modules/@swc/core`
`npm WARN   peerOptional @swc/core@">=1.2.50" from ts-node@10.3.0`
`npm WARN   node_modules/ts-node`
`npm WARN   2 more (@swc-node/register, @swc-node/core)`
`npm ERR! code ETARGET`
`npm ERR! notarget No matching version found for @swc/helpers@>0.4.14 <0.5.`
`npm ERR! notarget In most cases you or one of your dependencies are requesting`
`npm ERR! notarget a package version that doesn't exist.`

**Comment by jaward916** [comment]
I thought this was resolved by the 1.3.46 publishing (https://github.com/swc-project/swc/actions/runs/4606710328) but after several hours, and lots of cache clearing I cannot get past this error at all, has anyone found a way?
